pcdsdevices/lodcm.py

```diff
@@ -11,17 +11,23 @@
 import logging
 import functools
 
-from ophyd import Component as Cpt, FormattedComponent as FCpt
-from ophyd.signal import EpicsSignal, EpicsSignalRO
+from ophyd.device import Device, Component as Cpt
 from ophyd.sim import NullStatus
 from ophyd.status import wait as status_wait
 
-from .doc_stubs import basic_positioner_init, insert_remove
+from .doc_stubs import insert_remove
 from .inout import InOutRecordPositioner
 
 logger = logging.getLogger(__name__)
 
 
+class H1N(InOutRecordPositioner):
+    states_list = ['OUT', 'C', 'Si']
+    in_states = ['C', 'Si']
+    _states_alias = {'C': 'IN'}
+    _transmission = {'C': 0.8, 'Si': 0.7}
```

Comment:
The more I've seen these transmissions the more I dislike them. Think it might be worth just going to a binary of `transmissive` or not...

Reply:
Based on the comment below:
```
    # TBH these are guessed. Please replace if you know better. These don't
    # need to be 100% accurate, but they should reflect a reasonable reduction
    # in transmission.
```
I think @teddyrendahl may have something with the change to a boolean.

Reply:
Probably correct, for devices that are > 80% transmissive it isn't a relevant quantity. In the past this has mattered a lot for random devices that I didn't expect due to absorption edges, but that isn't even covered by ratios like these.